CONTRIBUTING.md

```
@@ -0,0 +1,259 @@
+|------|------------|---------|
+| Modules | `snake_case` | `tool_state.lua` |
+| Functions | `snake_case` | `create_tool()` |
+| Local variables | `snake_case` | `tool_name` |
+| Constants | `UPPER_SNAKE_CASE` | `INITIAL_STATE` |
+| Classes/Types | `PascalCase` | `ToolState` |
+
+## Commit Messages
+
+Follow [Conventional Commits](https://www.conventionalcommits.org/):
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)